crates/aptos-protos/proto/aptos/datastream/v1/datastream.proto

```diff
@@ -42,6 +42,9 @@ message StreamStatus {
 message RawDatastreamRequest {
   // Required; start version of current stream.
   uint64 starting_version = 1;
+
+  // Optional; end version of current stream, exclusive.
+  optional uint64 ending_version = 2;
```

Comment:
can we make this amount actually? Cuz ending_version is ambiguous. I think that it should be inclusive but some ppl might think otherwise. Remember we return an inclusive end version in the batch end status.

Reply:
done.